src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/wcm/templates/basic-af/initial/.content.xml

```diff
@@ -8,9 +8,12 @@
             jcr:lastModified="{Date}2021-03-24T18:22:20.809+05:30"
             sling:resourceType="fd/af/components/page2/aftemplatedpage"
             guideComponentType="fd/af/templates">
-        <parsys1
+        <container1
                 jcr:primaryType="nt:unstructured"
-                sling:resourceType="wcm/foundation/components/responsivegrid">
+                jcr:title="Header"
+                sling:resourceType="wcm/foundation/components/responsivegrid"
```

Comment:
I might be missing something here, but what is the change done here? The resourceType still seems to be the same. Only the tag name is changed rom parsys1 to container1, does this even make any difference? I believe its just a name